pyClient/zeth/joinsplit.py

```diff
@@ -697,12 +702,17 @@ def joinsplit_sign(
     """
     assert len(ciphertexts) == constants.JS_INPUTS
 
+    sender_eth_addr_bytes = encode_eth_address(sender_eth_address)
+    print(f"sender_eth_addr_bytes = {sender_eth_addr_bytes.hex()}")
+
     # The message to sign consists of (in order):
+    #   - senders Ethereum address
     #   - senders public encryption key
     #   - ciphertexts
     #   - proof elements
     #   - public input elements
     h = sha256()
+    h.update(encode_eth_address(sender_eth_address))
```

Comment:
Why don't you reuse `sender_eth_addr_bytes` defined just above ?

Reply:
Good catch.  Removed that variable (it was only for debugging).